app/policies/rent_policy.rb

```diff
@@ -0,0 +1,9 @@
+class RentPolicy < ApplicationPolicy
+  def index?
+    @record.present? && @user.present? && @record.map { |record| record.user == @user }.all?
```

Comment:
A better validation you could do is to use [scopes](https://github.com/varvet/pundit#scopes), that way you _scope_the list of rents that are returned to the user